src/pages/billing/List.js

```diff
@@ -297,6 +303,302 @@ const NewBillingProjectModal = ({ onSuccess, onDismiss, billingAccounts, loadAcc
+            labelStyle: { ...styles.radioButtonLabel },
+            disabled: !isActive,
+            onChange: () => {
+              if (!isDone) { next() }
+              setAccessToBillingAccount(false)
+            }
+          })
+        ]),
+        div({ style: { marginTop: '2rem', display: 'flex', displayDirection: 'row' } }, [
+          h(RadioButton, {
+            text: 'I have a billing account',
+            name: 'access-to-account',
+            checked: accessToBillingAccount === true,
+            labelStyle: { ...styles.radioButtonLabel },
+            disabled: !isActive,
+            onChange: () => {
+              if (!isDone) { next() }
+              setAccessToBillingAccount(true)
+            }
+          })
+        ])
+      ])
+    ])
+  }
+
+  const step3 = () => {
+    const isActive = activeStep === 3
+    const isDone = activeStep > 3
+
+    const haveAccessToBillingHeader =
+      div({ style: { width: '75%' } }, [
+        h2({ style: { fontSize: 18, marginTop: 0 } }, ['STEP 3']),
+        span({ style: { fontSize: 14, lineHeight: '22px', whiteSpace: 'pre-wrap' } },
+          ['Add ', span({ style: { fontWeight: 'bold' } }, 'terra-billing@terra.bio'), ' as a Billing Account User',
+            span({ style: { fontWeight: 'bold' } }, ' to your billing account.')]),
+        div({ style: { marginTop: '1rem' } }, [
+          h(Link, {
+            ...Utils.newTabLinkProps, style: { textDecoration: 'underline' },
+            href: 'https://support.terra.bio/hc/en-us/articles/360026182251'
+          }, [
+            'Learn how to set up a Google Cloud Billing account'
+          ])
+        ])
+      ])
+    const dontHaveAccessToBillingHeader =
+      div({ style: { width: '55%' } }, [
+        h2({ style: { fontSize: 18, marginTop: 0 } }, ['STEP 3']),
+        span({ style: { fontSize: 14, lineHeight: '22px', whiteSpace: 'pre-wrap' } }, [
+          'Contact your billing account administrator and have them add you and ',
+          span({ style: { fontWeight: 'bold' } }, ['terra-billing@terra.bio']), ' as a Billing Account User',
+          span({ style: { fontWeight: 'bold' } }, [' to your organization\'s billing account.'])
+        ]),
+        div({ style: { marginTop: '1rem' } }, [
+          h(Link, {
+            ...Utils.newTabLinkProps, style: { textDecoration: 'underline' },
+            href: 'https://support.terra.bio/hc/en-us/articles/360026182251'
+          }, [
+            'Learn how to set up a Google Cloud Billing account'
+          ])
+        ])
+      ])
+
+    return li({ style: { ...styles.stepBanner(isActive), flexDirection: 'row', alignItems: 'center', justifyContent: 'space-between' } }, [
+      isActive || isDone ?
+        (accessToBillingAccount ? haveAccessToBillingHeader : dontHaveAccessToBillingHeader) :
+        haveAccessToBillingHeader,
+      isActive || isDone ?
+        div({ style: { width: '25%', display: 'flex', displayDirection: 'row' } }, [
```

Comment:
You can't just have a single radio button-- it will need to be a checkbox.

![image](https://user-images.githubusercontent.com/484484/196791793-e68016fd-21ab-4ac6-95e6-602bb2384f33.png)